utils/ccomp.ml

```diff
@@ -147,17 +147,18 @@ let create_archive archive file_list =
         then r1
         else command(Config.ranlib ^ " " ^ quoted_archive)
 
-let expand_libname name =
-  if String.length name < 2 || String.sub name 0 2 <> "-l"
-  then name
-  else begin
-    let libname =
-      "lib" ^ String.sub name 2 (String.length name - 2) ^ Config.ext_lib in
-    try
-      Load_path.find libname
-    with Not_found ->
-      libname
-  end
+let expand_libname cclibs =
+  cclibs |> List.map (fun cclib ->
+    if String.length cclib < 2 || String.sub cclib 0 2 <> "-l"
```

Comment:
`not (String.starts_with ~prefix:"-l" cclib)`

Reply:
Good idea - I did `s/name/cclib` to match `remove_Wl`, but I didn't particularly look at the code itself!